src/backend/web/routes/posts.js

```diff
@@ -80,4 +81,27 @@ posts.get('/count', async (req, res) => {
   }
 });
 
+// Allow for the id to be a URI, possibly containing path separators
```

Comment:
This comment is now incorrect.  We can't include path separators anymore if using `:guid`.  Please change it to say:

```
// The guid is likely a URI, and must be encoded by the client.
```

Reply:
fixed.